api/src/download-replugged.ts

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+// For CLI usage
+// npm run download-replugged
+
+import { fetch } from "undici";
+import config from "./config.js";
+import { STORAGE_FOLDER, createDirForFile } from "./utils/misc.js";
+import path from "path";
+import { writeFile } from "fs/promises";
+
+const ADDONS_FOLDER = STORAGE_FOLDER("addons");
+
+const { token } = config.github;
+
+const headers: Record<string, string> = {};
+if (token) {
+  headers.Authorization = `token ${token}`;
```

Comment:
See previous comment about this.

```suggestion
  headers.Authorization = `Bearer ${token}`;
```